af04f855-58c4-4db3-a191-45fe33381679/scripts/actions.py

```diff
@@ -732,3 +758,35 @@ def moveOneRandom(group):
 	card.moveTo(me.hand)
 	notify("{} randomly moves {} from their discard to their hand.".format(me, card.name))
 	
+#------------------------------------------------------------------------------
+# Check Deck Tournament Validity
+#------------------------------------------------------------------------------
+
+def CheckTournamentValid(group, x=0, y=0):
+	LegacyLegal=["01-Core Set", "02-Warlord Cycle", "03-The Great Devourer", "04-Planetfall Cycle" "05-Legions of Death", "06-Deathworld Cycle"]
+	ApokaLegal=["01-Core Set", "02-Warlord Cycle", "03-The Great Devourer", "04-Planetfall Cycle" "05-Legions of Death", "06-Deathworld Cycle", "APK-01-Navida Prime Cycle","APK-02-Defenders of the Faith","APK-03-Confrontation Cycle"]
+	choiceList = ['Legacy (FFG)', 'Extended (FFG + Apoka)']
```

Comment:
Leave out Legacy/Extended and just call it FFG and FFG + Apoka, a third option could possibly be FFG + BCL